### ISSUE TITLE
Project Name?

### ISSUE THREAD

**Issue by cssagogo** [krystalcampioni/vue-hotel-datepicker#159]
I'm wondering why you limited the usage of this picker to hotels.  This date picker could be used for any type of UI requiring a start and end date picker.  For my case I need something like this for an analytics application.  I really like the look of this, but hesitate to use something called vue-hotel-datepicker in my project.

**Comment by Epistol** [comment]
Double that, it's one of the only good range picker for vue, but the whole night thing is a bit useless in others situation.

**Comment by matiasperrone** [comment]
Yeah... it's true... perharps we should you label instead of "price" right?....
I am not sure about the implications...

**Comment by matiasperrone** [comment]
@krystalcampioni I think that vue-range-datepicker should be a good idea? What do you think?

**Comment by krystalcampioni** [comment]
The project was named `vue-hotel-datepicker` just because that was the original usage of the library. Also, `vue-range-datepicker` is already taken, as most of the _good names_ 😅 

I think it might make sense to keep this one as `vue-hotel-datepicker` since it does have a lot of logic implemented to handle checkin/checkout, blocking dates etc. We could create a simpler version of it with fewer features and name it something else, though 🤔

**Comment by matiasperrone** [comment]
There is not a "https://www.npmjs.com/package/vue-range-datepicker"
